generator/uscore/uscore_generator.rb

```diff
@@ -200,14 +200,16 @@ def create_include_test(sequence)
       end
 
       def create_revinclude_test(sequence)
+        first_search = find_first_search(sequence)
+        return if first_search.blank?
+
         revinclude_test = {
-          tests_that: "Server returns the appropriate resources from the following _revincludes: #{sequence[:revincludes].join(',')}",
+          tests_that: "Server returns valid Provenance resources from #{sequence[:resource]} search by #{first_search[:names].join(' + ')} + _revIncludes: Provenance:target",
```

Comment:
This test doesn't validate the Provenance resources.

Reply:
removed 'valid' from description.